app/serializers/v1/customer_usage_serializer.rb

```diff
@@ -21,6 +19,14 @@ def serialize
 
     private
 
+    def currency
+      model.currency || model.amount_currency
```

Comment:
Do we need to keep this retro-compatibility?

Reply:
Only for a short period. Since usage is cached, I don't want to break it with a deploy. When all cached values will expires, we will be able to remove it safely. I will had a **TODO** comment not to forget it.